queries.sql

```diff
@@ -72,3 +72,101 @@ SELECT species,
     WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' 
     GROUP BY species;
 
+/* Multiple tables queries */
+SELECT name AS animal_names, full_name AS owner_name FROM animals 
+JOIN owners ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name AS animal_names, species.name AS animal_species FROM animals 
+JOIN species ON  animals.species_id = species.id 
+WHERE species.id = 1;
+
+SELECT full_name, name AS animal_names
+FROM animals 
+RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+SELECT species.name AS species_name, count(*) As species_count
+FROM species 
+JOIN animals ON species.id = animals.species_id 
+GROUP BY species.name;
+
+SELECT animals.* FROM animals
+LEFT JOIN owners
+ON animals.owner_id = owners.id
+LEFT JOIN species
+ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell'
+AND species.name = 'Digimon';
+
+SELECT name FROM animals 
+JOIN owners ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+
+SELECT owners.full_name AS full_name , COUNT(*) AS count 
+FROM animals JOIN owners ON animals.owner_id = owners.id 
+GROUP BY owners.full_name 
+ORDER BY count DESC LIMIT 1;
+
+/* Join Table */
+
+SELECT animals.name AS last_animal_seen_by_William_Tatcher
+FROM animals 
+JOIN visits ON animals.id = visits.animals_id 
+JOIN vets ON visits.vets_id = vets.id 
+WHERE vets.name='William Tatcher' 
+ORDER BY visits.visit_date DESC LIMIT 1;
+
+SELECT COUNT(*) AS no_of_different_animals_seen_by_Stephanie_Mendez  
+FROM (SELECT DISTINCT animals.name FROM animals 
+JOIN visits ON animals.id = visits.animals_id JOIN vets ON visits.vets_id = vets.id 
+WHERE vets.name='Stephanie Mendez') AS foo;
+
+SELECT vets.name AS vet_names, species.name AS vet_specialities
+FROM vets 
+FULL JOIN specialization ON vets.id = specialization.vets_id FULL 
+JOIN species ON specialization.species_id = species.id;
+
+SELECT animals.name AS animals_that_visited_Stephanie_Mendez
+FROM animals 
+JOIN visits ON animals.id = visits.animals_id 
+JOIN vets ON visits.vets_id = vets.id 
+WHERE vets.name='Stephanie Mendez' 
+AND visits.visit_date > '2019-04-01' 
+AND visits.visit_date <'2020-08-30';
```

Comment:
- [ ] awesome query, One thing left is we need to make sure is the first query should be `2020-04-01` instead of `2019-04-01` as the requirement is `between April 1st and August 30th, 2020`

Reply:
I have also made this required change and fixed the error in the visit date. 👍🏼